src/ssh/azext_ssh/_params.py

```diff
@@ -2,22 +2,31 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
+from azure.cli.core.commands.parameters import (
+    get_three_state_flag
+)
 
 
 def load_arguments(self, _):
 
     with self.argument_context('ssh vm') as c:
-        c.argument('vm_name', options_list=['--vm-name', '-n'], help='The name of the VM')
+        c.argument('vm_name', options_list=['--vm-name', '--name', '-n'], help='The name of the VM')
         c.argument('ssh_ip', options_list=['--ip'], help='The public IP address (or hostname) of the VM')
         c.argument('public_key_file', options_list=['--public-key-file', '-p'], help='The RSA public key file path')
         c.argument('private_key_file', options_list=['--private-key-file', '-i'], help='The RSA private key file path')
+        c.argument('use_private_ip', options_list=['--prefer-private-ip'],
+                   help='Will use a private IP if available. By default only public IPs are used.')
 
     with self.argument_context('ssh config') as c:
         c.argument('config_path', options_list=['--file', '-f'], help='The file path to write the SSH config to')
-        c.argument('vm_name', options_list=['--vm-name', '-n'], help='The name of the VM')
+        c.argument('vm_name', options_list=['--vm-name', '--name', '-n'], help='The name of the VM')
         c.argument('ssh_ip', options_list=['--ip'], help='The public IP address (or hostname) of the VM')
         c.argument('public_key_file', options_list=['--public-key-file', '-p'], help='The RSA public key file path')
         c.argument('private_key_file', options_list=['--private-key-file', '-i'], help='The RSA private key file path')
+        c.argument('use_private_ip', options_list=['--prefer-private-ip'],
+                   help='Will use a private IP if available. By default only public IPs are used.')
+        c.argument('overwrite', arg_type=get_three_state_flag(), options_list=['--overwrite'],
+                   help='Overwrites the config file if this flag is set')
```

Comment:
You may use `action='store_true'` instead of `arg_type=get_three_state_flag()` for `overwrite` as it's not a property but more like a behavior switch as explained in [Registering Flags](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags).

Reply:
Gotcha, changed.